learn/cloud/monitoring.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Monitoring metrics
+description: Meilisearch Cloud offers in-depth search analytics to help monitor your application performance.
+sidebarDepth: 3
+---
+
+# Monitoring
+
+Enable Meilisearch Cloud monitoring to keep track of application performance and service status. 
```

Comment:
I was trying to avoid mentioning any specific charts directly to minimise future maintenance. For example, if we mention e.g. search latency but eventually decide to move it to analytics or even remove it completely, we then need to update this intro.

If you still think it's important, could you single out a couple of graphs you expect users to find particularly useful?

Reply:
We could do that as a future iteration @guimachiavelli; I think that always beneficial to show what value they could extract from it